File: Sorting/bubble-sort.js
Array.prototype.swap = function (x,y) {
  var b = this[x];
  this[x] = this[y];
  this[y] = b;
  return this;
}

let array = [-1, 0, -100, 11, 10, 19, 1 , 50, 11, 7];

for(let i = 1; i < array.length; ++i) {
  for(let j = 0 ; j < array.length - i ; ++j) {
    if(array[j] > array[j+1]) {
      array.swap(j, j+1);
    }
  }
}

console.log(array);

File: Sorting/bubblesort.py
def bubbleSort(arr):
    n = len(arr)
 
    for i in range(n):
 
        # Last i elements are already in place
        for j in range(0, n-i-1):
 
            # traverse the array from 0 to n-i-1
            # Swap if the element found is greater
            # than the next element
            if arr[j] > arr[j+1] :
                arr[j], arr[j+1] = arr[j+1], arr[j]
 
n=int(input("Enter no.of elements"))
for i in range(n):
    arr[i]=int(input())

bubbleSort(arr)
 
print ("Sorted array is:")
for i in range(len(arr)):
    print ("%d" %arr[i]), 